### PR TITLE
Fixed #3725 - Localization strings are missing for survey element properties

### LINKED ISSUE #3725: Localization strings are missing for survey element properties
Translation items are seem to be missing for the the following properties:
- [ ] An Image Question: Image Fit values (Contain | Cover | Fill)
image

- [ ] Ranking: the  `Comment placeholder` property
image

- [ ] Checkbox: 
* `Use display values in dynamic texts`
image
- `Clear if invisible`  
image

### PR DESCRIPTION
**Pull request by tsv2013**

### CODE REVIEW

File: packages/survey-creator-core/src/localization/english.ts [hunk 1]
```diff
@@ -661,6 +656,16 @@ export var enStrings = {
     triggerRunExpressionEmpty: "Please enter a valid expression",
     emptyExpressionPlaceHolder: "Type expression here...",
     noFile: "No file choosen",
+    clearIfInvisible: "Clear if invisible",
+    valuePropertyName: "Value property name",
+    searchEnabled: "Search enabled",
```
Comment (RomanTsukanov) [inline packages/survey-creator-core/src/localization/english.ts:661]:
```suggestion
    searchEnabled: "Enable search",
```

File: packages/survey-creator-core/src/localization/english.ts [hunk 2]
```diff
@@ -661,6 +656,16 @@ export var enStrings = {
...
+    hideSelectedItems: "Hide selected items",
+    signatureWidth: "Signature width",
+    signatureHeight: "Signature height",
+    verticalAlign: "Vertical align",
```
Comment (RomanTsukanov) [inline packages/survey-creator-core/src/localization/english.ts:665]:
```suggestion
    verticalAlign: "Vertical alignment",
```

File: packages/survey-creator-core/src/localization/english.ts [hunk 3]
```diff
@@ -661,6 +656,16 @@ export var enStrings = {
...
+    alternateRows: "Alternate rows",
+    columnsVisibleIf: "Columns visible if",
```
Comment (RomanTsukanov) [inline packages/survey-creator-core/src/localization/english.ts:667]:
```suggestion
    columnsVisibleIf: "Columns are visible if",
```

File: packages/survey-creator-core/src/localization/english.ts [hunk 4]
```diff
@@ -661,6 +656,16 @@ export var enStrings = {
...
+    rowsVisibleIf: "Rows visible if"
```
Comment (RomanTsukanov) [inline packages/survey-creator-core/src/localization/english.ts:668]:
```suggestion
    rowsVisibleIf: "Rows are visible if"
```

File: packages/survey-creator-core/src/localization/english.ts [hunk 5]
```diff
@@ -661,6 +656,16 @@ export var enStrings = {
     triggerRunExpressionEmpty: "Please enter a valid expression",
     emptyExpressionPlaceHolder: "Type expression here...",
     noFile: "No file choosen",
+    clearIfInvisible: "Clear if invisible",
```
Comment (RomanTsukanov) [inline packages/survey-creator-core/src/localization/english.ts:659]:
```suggestion
    clearIfInvisible: "Clear the value if the question becomes hidden",
```

File: packages/survey-creator-core/src/localization/english.ts (hunk 2 above)
Comment (tsv2013) [reply packages/survey-creator-core/src/localization/english.ts]: done

File: packages/survey-creator-core/src/localization/english.ts (hunk 3 above)
Comment (tsv2013) [reply packages/survey-creator-core/src/localization/english.ts]: done

File: packages/survey-creator-core/src/localization/english.ts (hunk 4 above)
Comment (tsv2013) [reply packages/survey-creator-core/src/localization/english.ts]: done

File: packages/survey-creator-core/src/localization/english.ts (hunk 5 above)
Comment (tsv2013) [reply packages/survey-creator-core/src/localization/english.ts]: done

**Review by RomanTsukanov — APPROVED** [review]

**Pull request merged**